### PR TITLE
add apache config

### PR DESCRIPTION
**Pull request by mnecas**
add logic to control apache config 
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1772091
@mwperina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,12 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_provider_ovn_username | admin@internal     | Username for OVN. |
 | ovirt_engine_setup_provider_ovn_password | UNDEF              | Password for OVN. |
 
+* Apache releated options:
```
Comment (mwperina) [inline README.md]: releated -> related

File: README.md [hunk 2]
```diff
@@ -93,6 +93,12 @@ as ``ovirt_engine_setup_answer_file_path``.
...
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_apache_config_root_redirection | true               | If `False` the apache will not be set as default page of the web server.   |
```
Comment (mwperina) [inline README.md]: true -> True

Comment (mwperina) [reply README.md]: Description: If `True` engine-setup will configure the default page in Apache to automatically redirect clients to ovirt-engine default page.

File: README.md [hunk 3]
```diff
@@ -93,6 +93,12 @@ as ``ovirt_engine_setup_answer_file_path``.
...
+| ovirt_engine_setup_apache_config_ssl | true     | If `False` ssl will need to be configured manually. |
```
Comment (mwperina) [inline README.md]: true -> True

Comment (mwperina) [reply README.md]: Description: If 'False', engine-setup will not configure Apache SSL settings, so administrators will need to configure them manually

File: templates/answerfile_4.4_basic.txt.j2 [hunk 4]
```diff
@@ -1 +1,7 @@
 {% include "./templates/answerfile_4.3_basic.txt.j2" %}
+{% if ovirt_engine_apache_config_root_redirection is defined %}
+QUESTION/1/OVESETUP_APACHE_CONFIG_ROOT_REDIRECTION=str:{{ ovirt_engine_apache_config_root_redirection | ternary('yes','no')  }}
+{% endif %}
+{% if ovirt_engine_apache_config_ssl is defined %}
+QUESTION/1/OVESETUP_APACHE_CONFIG_SSL=str:{{ ovirt_engine_apache_config_ssl | ternary('automatic','manual') }}
+{% endif %}
```
Comment (mwperina) [inline templates/answerfile_4.4_basic.txt.j2]: Can we add above changes to 4.3 template, so it's available for both 4.3 and 4.4?

Comment (mnecas) [reply templates/answerfile_4.4_basic.txt.j2]: Moved the changes to answerfile_4.3 so it will be compatible with 4.3 and 4.4

**Pull request merged**